src/fidesops/ops/service/email/email_dispatch_service.py

```diff
@@ -26,10 +24,7 @@ def dispatch_email(
     db: Session,
```

Comment:
Nice work on this broader email service you implemented earlier; it's worked well for all of the use cases we've added lately.

Reply:
this makes me so happy to hear!